examples/05_unit_input_data.py

```diff
@@ -35,7 +35,7 @@
 
     # Create a new session. No need to try/except this: if session creation
     # fails, there is not much we can do anyways...
-    session = rp.Session()
+    session = rp.Session(uid='TarTest')
```

Comment:
Please don't hardcode session names in the examples - this will fail on the second execution.

Reply:
Done